src/cap32.cpp

```diff
@@ -1220,6 +1221,9 @@ void bin_load (const std::string& filename, const size_t offset)
     return;
   }
 
+  auto closure = [&]() { fclose(file); };
+  memutils::deref<decltype(closure)> cs(closure); // TODO: when C++20, can become a one liner expression.
```

Comment:
What needs to change in C++20 to have the one liner?

Reply:
well with C++ 20 you can simplify like
```
memutils::scope_exit cs([&](){ fclose(file); });
```